### ISSUE TITLE
Android falied to load config with valid server ssl certificate

### ISSUE THREAD

**Issue by pinkrickshaw** [jitsi/jitsi-meet#3251]
Hello we are working with jitsi and successfully setup it on our server we have valid ssl certificate and  we have develop a fully functional conference system which is working but now when we clone our app for new project we got this error 

ReactNativeJS: 'Failed to load config from https://---domain-name---/config.js?room=rahul', [TypeError: Network request failed]
07-13 16:26:01.334 3900-3949/com.vc.prd.confab E/unknown:React: console.error: "Failed to load config from https://---domain-name---/config.js?room=rahul", {}, stack:
    <unknown>@401:2404
    <unknown>@489:602
    u@211:133
    <unknown>@211:919
    <unknown>@86:966
    callTimer@81:601
    callImmediatesPass@81:1746
    callImmediates@81:1862
    <unknown>@79:2436
    <unknown>@79:106
    value@79:2401
    <unknown>@79:1039
    <unknown>@79:106
    value@79:985

Its works fine in our one app but not works now when we try to clone it or making new project we got this error .This is even not work in any other app has only one activity has only video jitsi code.

**Comment by CPLASF1925** [comment]
@paweldomas 

Android client to access their own servers how to configure, and where to add certificates to their own certificates.

**Comment by CPLASF1925** [comment]
@pinkrickshaw  https://kanavkahol.com 

Where to add a certificate

**Comment by ds88997** [comment]
Hello we are able to access config.js file in mobile web browser but app gives error or reconnecting with in 30 seconds or network error seems again.I am from Pinkrickshaw above conversation is raise by me.

**Comment by ds88997** [comment]
> Are you able to open https://kanavkahol.com/cofnig.js in a web browser on this phone ?

Hello we are able to access config.js file in mobile web browser but app gives error or reconnecting with in 30 seconds or network error seems again.I am from Pinkrickshaw above conversation is raise by me.

**Comment by saghul** [comment]
Can you test the Jitsi Meet app and see if you get the same behavior?

**Comment by ds88997** [comment]
Jitsi meet app work fine but on meet.jit.si  but when i switch to my own server url it shows reconnecting error but it works fine for web browser and also on mobile browser but not in app
![screenshot_20181112-161339](https://user-images.githubusercontent.com/31955686/48342888-e702bf80-e696-11e8-9689-4f6459495052.png)

error log is


'Failed to load config from https://kanavkahol.com/config.js?room=room1', [TypeError: Network request failed]
2018-11-12 16:19:53.262 16396-16646/feed.pink.com.video E/unknown:ReactNative: console.error: "Failed to load config from https://kanavkahol.com/config.js?room=room1", {}, stack:
    <unknown>@332:2579
    <unknown>@722:1142
    <unknown>@66:629
    <unknown>@66:657
    <unknown>@40:1772
    b@40:628
    w@40:992
    callImmediates@40:3173
    value@35:2778
    <unknown>@35:1360
    value@35:2606
    value@35:1330
    value@35:938
2018-11-12 16:19:54.394 1029-1042/? E/RemotePrintSpooler: servicesToKeep is empty!
2018-11-12 16:19:57.653 1823-18165/? E/NetworkScheduler: Invalid component specified.
2018-11-12 16:20:02.449 1029-1046/? E/BatteryStatsService: no controller energy info supplied
2018-11-12 16:20:02.449 1029-1046/? E/BatteryStatsService: no controller energy info supplied
2018-11-12 16:20:02.473 2312-2312/? E/MtpServerJNI: server is null in send_object_removed
2018-11-12 16:20:10.255 1029-1178/? E/ConnectivityService: RemoteException caught trying to send a callback msg for NetworkRequest [ id=87, legacyType=-1, [ Capabilities: INTERNET&NOT_RESTRICTED&TRUSTED] ]
2018-11-12 16:20:12.683 1029-1046/? E/BatteryStatsService: power: Missing API
2018-11-12 16:20:12.685 1029-1046/? E/BatteryStatsService: no controller energy info supplied
2018-11-12 16:20:12.685 1029-1046/? E/BatteryStatsService: no controller energy info supplied
2018-11-12 16:20:12.726 2312-2312/? E/MtpServerJNI: server is null in send_object_removed
2018-11-12 16:20:15.926 2572-2706/? E/ContactsProvider_EventLog: Flush buffer to file cnt : 1 size : 0Kb duration : 8ms lastUpdatedAfter : 60110 ms mFlush_time_threasold : 2000 mCurrentSize : 354

**Comment by saghul** [comment]
I can reproduce the problem. Works ok on iOS, but I'm getting a black screen on Android, not even the retry screen. I'll take a look as soon as I can, but it may take a bit of time, sorry.

**Comment by ds88997** [comment]
> I can reproduce the problem. Works ok on iOS, but I'm getting a black screen on Android, not even the retry screen. I'll take a look as soon as I can, but it may take a bit of time, sorry.

Thanks for your response one thing we are not working on IOS i am saying on "Website" . and it sounds good waiting for your response.

**Comment by saghul** [comment]
Yeah, sure, I just did the extra test of checking on iOS to check if it worked there.

**Comment by ds88997** [comment]
Hello We resolve our issue ,it was our server did not have ssl chain certificate installed so we just completed the chain by adding intermediate certificate provided by our certificated provider and its work smoothly.Thanks for your support and response.

**Comment by Amir-yazdanmanesh** [comment]
Can i run Jitsi with ignore SSL problems?
